src/miner/[minerId].md

```diff
@@ -0,0 +1,71 @@
+```js
+import { LineGraph } from "../components/line-graph.js";
+import { getDateXDaysAgo } from "../utils/date-utils.js";
+const data = FileAttachment(`../data/${observable.params.minerId}-spark-rsr-summary.json`).json();
+```
+
+<div class="hero">
+  <body><a href="/"><img src="../media/spark-logomark-blue-with-bbox.png" alt="Spark Logo" width="300" /></a><body>
+    <h2>Dashboard Beta</h2>
+    <body><a href="https://filspark.com/dashboard" target="_blank" rel="noopener noreferrer">(Click here for Legacy Spark Grafana Dashboard)</a><body>
+</div>
+
+<h4>Miner Spark RSR Summary</h4>
+<body>This section shows the miner Spark Retrieval Success Rate Score summary. You can adjust the date range. Records start on the 7th April 2024.</body>
+
+
+```js
+const start = view(Inputs.date({label: "Start", value: getDateXDaysAgo(180) }));
```

Comment:
```suggestion
const start = view(Inputs.date({label: "Start", value: "2024-04-07" }));
```

I think it makes sense to show all data by default

Reply:
But I'm not sure

Reply:
I've seen that main dashboard shows data for the past 180 days so I went with it.